sdk/synapse/azure-synapse-artifacts/azure/synapse/artifacts/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.18.1"
+VERSION = "0.18.0"
```

Comment:
0.19.0
nit: if you change package-version of `swagger/README.md`, the codegen will help update it.